From 6cd37fa2dcbc6ec3def646c1bae67fd26873a77c Mon Sep 17 00:00:00 2001
From: Mingcong Bai <baimingcong@uniontech.com>
Date: Thu, 18 Jul 2024 11:42:28 +0800
Subject: [PATCH 168/360] arch: x86: configs: enable all Zhaoxin features

- Enable Zhaoxin features introduced by vendor patches.
- Disable KX-7000 pinctrl for now as the driver code is broken.

Signed-off-by: Mingcong Bai <baimingcong@uniontech.com>
---
 arch/x86/configs/deepin_x86_desktop_defconfig | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/x86/configs/deepin_x86_desktop_defconfig b/arch/x86/configs/deepin_x86_desktop_defconfig
index 3715f91b2d56..75d885d05496 100644
--- a/arch/x86/configs/deepin_x86_desktop_defconfig
+++ b/arch/x86/configs/deepin_x86_desktop_defconfig
@@ -2175,6 +2175,7 @@ CONFIG_PINCTRL_METEORLAKE=m
 CONFIG_PINCTRL_SUNRISEPOINT=m
 CONFIG_PINCTRL_TIGERLAKE=m
 CONFIG_PINCTRL_MERRIFIELD=m
+# CONFIG_PINCTRL_KX7000 is not set
 CONFIG_GPIO_SYSFS=y
 CONFIG_GPIO_AMDPT=m
 CONFIG_GPIO_DWAPB=m
@@ -2542,6 +2543,7 @@ CONFIG_SENSORS_TMP421=m
 CONFIG_SENSORS_TMP464=m
 CONFIG_SENSORS_TMP513=m
 CONFIG_SENSORS_VIA_CPUTEMP=m
+CONFIG_SENSORS_ZHAOXIN_CPUTEMP=m
 CONFIG_SENSORS_VIA686A=m
 CONFIG_SENSORS_VT1211=m
 CONFIG_SENSORS_VT8231=m
@@ -5515,6 +5517,7 @@ CONFIG_CRYPTO_DES3_EDE_X86_64=m
 CONFIG_CRYPTO_SERPENT_SSE2_X86_64=m
 CONFIG_CRYPTO_SERPENT_AVX2_X86_64=m
 CONFIG_CRYPTO_SM4_AESNI_AVX2_X86_64=m
+CONFIG_CRYPTO_SM4_ZHAOXIN_GMI=m
 CONFIG_CRYPTO_TWOFISH_AVX_X86_64=m
 CONFIG_CRYPTO_ARIA_GFNI_AVX512_X86_64=m
 CONFIG_CRYPTO_AEGIS128_AESNI_SSE2=m
@@ -5525,6 +5528,7 @@ CONFIG_CRYPTO_SHA1_SSSE3=m
 CONFIG_CRYPTO_SHA256_SSSE3=m
 CONFIG_CRYPTO_SHA512_SSSE3=m
 CONFIG_CRYPTO_SM3_AVX_X86_64=m
+CONFIG_CRYPTO_SM3_ZHAOXIN_GMI=m
 CONFIG_CRYPTO_GHASH_CLMUL_NI_INTEL=m
 CONFIG_CRYPTO_CRC32C_INTEL=y
 CONFIG_CRYPTO_CRC32_PCLMUL=m
-- 
2.20.1

